src/KeyVault/KeyVault/help/Set-AzKeyVaultAccessPolicy.md

```diff
@@ -570,6 +572,10 @@ The acceptable values for this parameter:
 - backup
 - restore
 - purge
+- setsas
```

Comment:
The values are generated from the https://github.com/Azure/azure-powershell/blob/f7955797a4f5106a909e78e5e0f95db0d1b27134/src/KeyVault/KeyVault/Commands/SetAzureKeyVaultAccessPolicy.cs#L403, please update the code and re-generate the doc